### DISCUSSION TITLE
OP Crosh didn't work

### DISCUSSION THREAD

**Discussion by doofenator** [3kh0/ext-remover#770] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
So i did OP crosh and chose to only disable the blocker extension because I need ublock origin for javascript running but the blocker just reinstalled itself, help and lmk if theres a way to remove extension folders

**Comment by 3kh0** [comment, marked as answer]
Make sure you are running all the commands in the proper order, you might have messed something up somehow, screenshots pls

**Reply by doofenator** [reply]
k ill do it again
